packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -296,43 +304,52 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 			// https://dev.azure.com/fluidframework/internal/_workitems/edit/779
 			await this.lockedSummaryAction(
 				() => {},
-				async () =>
-					this.refreshLatestSummaryAckCallback({
+				async () => {
```

Comment:
Unrelated to this change - `handleSummaryAck()` should accept the `IAckedSummary` as a parameter. The call site `processIncomingSummaryAcks` assumes that the ack it's processing will the same that `handleSummaryAck` gets by calling `this.summaryCollection.latestAck` but it may have changed if an ack came in later.

Reply:
Totally agree, the way we handle summary acks in general should be updated. There's definitely a lot of bloat here.